enterprise/security/src/test/java/org/neo4j/server/security/enterprise/auth/ConfiguredAuthScenariosInteractionTestBase.java

```diff
@@ -80,4 +97,60 @@ public void shouldAllowRoleCallCreateNewTokensProceduresWhenConfigured() throws
         assertEmpty( noneSubject, "CALL db.createRelationshipType('MySpecialRelationship')" );
         assertEmpty( noneSubject, "CALL db.createProperty('MySpecialProperty')" );
     }
+
+    @Test
+    public void shouldWarnWhenUsingNativeAndOtherProvider() throws Throwable
+    {
+        configuredSetup( stringMap( SecuritySettings.auth_providers.name(), "native ,LDAP" ) );
+        assertSuccess( adminSubject, "CALL dbms.security.listUsers", r -> assertKeyIsMap( r, "username", "roles", userList ) );
+        GraphDatabaseFacade localGraph = neo.getLocalGraph();
+        InternalTransaction transaction = localGraph
+                .beginTransaction( KernelTransaction.Type.explicit, StandardEnterpriseSecurityContext.AUTH_DISABLED );
+        Result result =
+                localGraph.execute( transaction, "EXPLAIN CALL dbms.security.listUsers", Collections.emptyMap() );
+        assertThat(
+                containsNotification( result, "Native user management procedures will not affect non-native users." ),
+                equalTo( true ) );
+        transaction.success();
+        transaction.close();
+    }
+
+    @Test
+    public void shouldNotWarnWhenUsingNativeAndOtherProvider() throws Throwable
```

Comment:
Yes, wrong name on test.
There is no reason to test for when we don't have native configured, since the procedure is only available when one of the authproviders is the native.